test/e2e/network/dns.go

```diff
@@ -33,15 +33,16 @@ import (
 const dnsTestPodHostName = "dns-querier-1"
 const dnsTestServiceName = "dns-test-service"
 
-var _ = SIGDescribe("DNS", func() {
+func dnsTests(isIPv6 bool) {
 	f := framework.NewDefaultFramework("dns")
 
 	/*
 		Release : v1.9
-		Testname: DNS, cluster
+		Testname: DNS, cluster,
 		Description: When a Pod is created, the pod MUST be able to resolve cluster dns entries such as kubernetes.default via DNS and /etc/hosts.
 	*/
```

Comment:
The comment of line 39-43 is for conformance test.
We need to move the comment to line 74 if IPv4's one only is a conformance test, I think.

Reply:
@fturib this still needs to be addressed